_posts/2017-04-30-React-Native-for-iOS-devs.markdown

```diff
@@ -0,0 +1,671 @@
+---
+layout: epic
+title: Intro to React Native for an iOS Developer
+date: 2017-05-30
+categories: [Technology, emission, react-native, react, javascript]
+author: orta
+series: React Native at Artsy
+css: what-is-react-native
+---
+
+React Native is a new native library that vastly changes the way in which you can create applications. The majority of the information and tutorials on the subject come from the angle of _"you are a web developer, and want to do native"_.
+
+This makes sense, given that the size of that audience is much bigger, and far more open in the idea of writing apps using JavaScript. For web developers it opens a new creative space to work, however for native developers it provides a way to work with different tools on the same problem. Considering that most developers with a few years on the platform will be comfortable with the Xcode toolset, recommending a change this drastic is a tough sell.
```

Comment:
> audience is much bigger

bidder than what? I know, but being explicit can help frame the rest of the post.